metals/src/main/scala/scala/meta/metals/MetalsLanguageServer.scala

```diff
@@ -163,8 +171,21 @@ class MetalsLanguageServer(
           val folderPaths = folders.map(_.path)
 
           setupJna()
+
+          val folderPathsWithScala =
```

Comment:
Don't we want to use this in `scribe.info` and for removing old reports below instead of `folderPaths`?